src/firmware_new/boards/frankie_v1/frankie_v1.ioc

```diff
@@ -1,635 +1,639 @@
 #MicroXplorer Configuration settings - do not modify
-LWIP0.BSP.mode=
-FREERTOS.Queues01=RobotLogProtoQ,16,TbotsProto_RobotLog,0,Dynamic,NULL,NULL
-LWIP.LWIP_MULTICAST_TX_OPTIONS=0
-PB13.GPIOParameters=GPIO_Label
-RCC.USART1Freq_Value=108000000
-RCC.TIM17Freq_Value=72000000
-RCC.PLLCLKFreq_Value=216000000
-PB14.GPIO_Label=LD3 [Red]
-PG6.GPIO_Label=USB_PowerSwitchOn [STMPS2151STR_EN]
-RCC.PLLQCLKFreq_Value=48000000
-PD3.GPIOParameters=GPIO_Label
-LWIP.LWIP_IPV6_DUP_DETECT_ATTEMPTS=3
-PD8.Locked=true
-RCC.RTCFreq_Value=32000
-PA3.GPIOParameters=GPIO_Label
+CORTEX_M7.AccessPermission-Cortex_Memory_Protection_Unit_Region0_Settings=MPU_REGION_FULL_ACCESS
```

Comment:
:warning: I opened this ioc in STM32CubeMX 6.2.0 and clicked migrate

Reply:
that seemed to actually work surprisingly well, version 6.2.0 codegen seems to be better, none of the code in user sections got wiped